packages/react/src/components/Button/Button.module.css

```diff
@@ -89,7 +89,7 @@
 
 .button.onlyIcon {
     padding: 0.5rem;
-    width: var(--button-size);
+    height: var(--button-size);
```

Comment:
hmm, I think this line can be removed all together since `height` is already set to be this at L18.

Reply:
Nice! Yeah, I couldn't see anything either, most important seems to be the padding in this use-case.